tests/fixtures/analysis/nallo/multiqc_data.json

```diff
@@ -0,0 +1,33 @@
+{
+    "report_data_sources": {},
+    "report_general_stats_data": [
+        {
+            "ADM1": {
+                "mean_coverage": 32.18,
+                "min_coverage": 0.0,
+                "max_coverage": 63906.0,
+                "coverage_bases": 99366056494,
+                "length": 3088286377,
+                "1_x_pc": 94.0,
+                "5_x_pc": 94.0,
+                "10_x_pc": 93.0,
+                "30_x_pc": 65.0,
+                "50_x_pc": 2.0,
+                "median_coverage": 13
```

Comment:
to pass the tests this has to be higher than the threshold, so for example 43

Reply:
thanks, I have increased it now.